e2e/tests/examples/sign-in.spec.ts

```diff
@@ -0,0 +1,70 @@
+import ProfilePage from 'app/page/profile-page';
+import { signIn, signInWithAccessToken, signOut } from 'utils/test-utils';
+import Navigation, { NavLink } from 'app/component/navigation';
+import { withPage, withSignIn } from 'libs/page-manager';
+import { config } from 'resources/workbench-config';
+import HomePage from 'app/page/home-page';
+import GoogleLoginPage from 'app/page/google-login';
+import CookiePolicyPage from 'app/page/cookie-policy';
+import WorkspacesPage from 'app/page/workspaces-page';
+
+describe('login tests', () => {
+  test('withPage: Sign in by access token', async () => {
+    await withPage()(async (page) => {
+      await signInWithAccessToken(page);
+
+      // Navigate to Profile page.
+      await Navigation.navMenu(page, NavLink.PROFILE);
+      const profilePage = new ProfilePage(page);
+      await profilePage.waitForLoad();
+      expect(await profilePage.isLoaded()).toBe(true);
+
+      await signOut(page);
+    });
+  });
+
+  test('withPage example: Sign in as READER', async () => {
+    await withPage()(async (page) => {
```

Comment:
I couldn't get `test('', withPage()(async (page, browser) => { ...})` syntax to compile.

Reply:
What was the error?

Reply:
<img width="859" alt="Screen Shot 2021-06-30 at 6 28 34 PM" src="https://user-images.githubusercontent.com/35533885/124039537-1e9e1180-d9d1-11eb-9c19-d74f76450dca.png">